### ISSUE TITLE
University Contains 2 "Unlocks Composter Huts" Researches

### ISSUE THREAD

**Issue by RobMarsh1201** [ldtteam/minecolonies#6947]
### Prerequisites 

- [x] I am running the latest **alpha** version of MineColonies and Structurize for my Minecraft version.
- [x] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure my issue is not covered there.
- [x] I made sure that **this issue is not a duplicate of any existing issue.**

 Context: minecolonies-0.13.684-BETA-universal.jar

- Minecraft Version: 1.16.5
- MineColonies Version: 0.13.684
- Structurize Version: structurize-0.13.148-ALPHA
- Related Mods and their Versions: Life in the village 2 modpack

## Expected behavior
The technology called "Let it Grow" should contain the following:

Let It Grow | Biodegradable | Farm at least level 3; 16 compost | Unlocks the Plantation

as per the [https://wiki.minecolonies.ldtteam.com/source/systems/research#technolog](Wiki.) 

## Actual behavior
The same string from the previous tech, "Biodegradable" is displayed.

![bugReport](https://user-images.githubusercontent.com/8396497/115117554-31c24900-9f6d-11eb-8412-db2cac1a51be.PNG)

## Steps to reproduce the issue

1. Open the Technology Research tree in the University.
2. Observe the "Let it grow" tech, and the prior one.

### Logs

N/A

**Comment by RobMarsh1201** [comment]
I have yet to test whether the tech is still functional or not. Will report back.

**Comment by gattsuru** [comment]
In 0.13.684.  The tech 'functions', but both unlock the Composter, and the Plantation didn't require an unlock.  I derped in datagen

This was fixed in https://github.com/ldtteam/minecolonies/pull/6645 ; I'd recommend updating for the other improvements.

If playing Life In The Village 2 on a server, there's an interaction between Apotheosis and Farmer's Delight; until Apotheosis updates, adding FastWorkbench fixes it.  Not required if single-player.
